dash-renderer/src/actions/dependencies.js

```diff
@@ -1247,11 +1247,13 @@ export function getCallbacksInLayout(graphs, paths, layoutChunk, opts) {
         if (outIdCallbacks) {
             for (const property in outIdCallbacks) {
                 const cb = getCallbackByOutput(graphs, paths, id, property);
-                // callbacks found in the layout by output should always run,
-                // ie this is the initial call of this callback even if it's
-                // not the page initialization but just a new layout chunk
-                cb.initialCall = true;
-                addCallback(cb);
+                if (cb) {
```

Comment:
Fix for the "can't set initialCall of false" that @chriddyp saw in https://github.com/plotly/dash/pull/1103#issuecomment-606326673

Turns out I had been mainly testing using different keys in the ID dict for different classes of objects - which, given the way I implemented this, is slightly more performant, but that shouldn't be a big deal and keeping the keys consistent is probably more intuitive for users - but we should make clear at least in our examples that this isn't necessary.